ch05.adoc

```diff
@@ -183,13 +187,16 @@ _This section has been superseded by the treatment of time series as a type of d
 [[grid-mappings-and-projections, Section 5.6, "Horizontal Coordinate Reference Systems, Grid Mappings, and Projections"]]
 === Horizontal Coordinate Reference Systems, Grid Mappings, and Projections
 
-When the coordinate variables for a horizontal grid are not longitude and latitude, it is required
-that the true latitude and longitude coordinates be supplied via the **`coordinates`** attribute.  A
-__grid mapping variable__ is required if, in addition, it is desired to describe the mapping between the
-given coordinate variables and the true latitude and longitude coordinates, or to describe the
```

Comment:
Want to change this to something like:
> When the coordinate variables for a horizontal grid are not longitude and latitude, it is recommended that the true latitude and longitude coordinates be supplied via the **`coordinates`** attribute for software that does not implement horizontal coordinate transformations. However, 2D longitude and latitude variables can become too large and practically burdensome and are not strictly required for this reason.

Reply:
@dblodgett-usgs I don't think we should get into what software might or might not do.

Reply:
I feel pretty strongly that we should provide a recommendation and the logic for it. Modified my suggested text a bit. In generally, I don't think we should get into how implementations work with CF but in this case, it is useful?

Reply:
See my proposed change below. Is that satisfactory?

Reply:
Yes. I am happy with this.